tensorboard/backend/experiment_id_test.py

```diff
@@ -76,6 +76,14 @@ def test_with_experiment_sub_path(self):
     response = self.server.get("/experiment/123/x/y")
     self._assert_ok(response, eid="123", path="/x/y", script="/experiment/123")
 
+  def test_with_empty_experiment_root_path(self):
```

Comment:
Could you also document the `self.server.get("/experiment/")` case?

Reply:
Good point; yes. Done.